libs/hmpb/src/render_ballot.test.tsx

```diff
@@ -76,3 +79,38 @@ test('renderMinimalBallotsToCreateElectionDefinition', async () => {
     );
   expect(electionDefinition).toEqual(fixtureElectionDefinition);
 });
+
+test(
+  'v3-compatible NH ballot',
+  async () => {
+    const fixtureElection = electionGeneralFixtures.readElection();
+    const allBallotProps = allBaseBallotProps(fixtureElection);
+    const renderer = await createPlaywrightRenderer();
+    const electionDefinition =
+      await renderMinimalBallotsToCreateElectionDefinition(
+        renderer,
+        ballotTemplates.NhBallotV3,
+        allBallotProps,
+        'vxf'
+      );
+
+    // Bubbles and WIA crop should be snapped to grid
+    for (const gridLayout of electionDefinition.election.gridLayouts!) {
+      for (const gridPosition of gridLayout.gridPositions) {
+        expect(gridPosition.column % 1).toEqual(0);
+        expect(gridPosition.row % 1).toEqual(0);
+      }
+      expect(gridLayout.optionBoundsFromTargetMark.bottom % 1).toEqual(0);
+      expect(gridLayout.optionBoundsFromTargetMark.left % 1).toEqual(0);
+      expect(gridLayout.optionBoundsFromTargetMark.right % 1).toEqual(0);
+      expect(gridLayout.optionBoundsFromTargetMark.top % 1).toEqual(0);
+    }
+
+    // Election date should be off by one day to account for timezone bug in v3
+    expect(fixtureElection.date.toISOString()).toEqual('2020-11-03');
+    expect(electionDefinition.election.date.toISOString()).toEqual(
+      '2020-11-04'
+    );
+  },
+  { timeout: 30_000 }
```

Comment:
Just curious, what run duration were you seeing when running this test?

Reply:
Last CI run says:
```
 ✓ src/render_ballot.test.tsx > v3-compatible NH ballot 14303ms
```